srtcore/epoll.cpp

```diff
@@ -501,16 +522,28 @@ int CEPoll::wait(const int eid, set<SRTSOCKET>* readfds, set<SRTSOCKET>* writefd
                         ++total;
                 }
 
-                ed.checkEdge(it); // NOTE: potentially erases 'it'.
+                IF_HEAVY_LOGGING(debug_sockets << " " << it->fd << ":"
+                        << IF_DIRNAME(it->events, SRT_EPOLL_IN, "R")
+                        << IF_DIRNAME(it->events, SRT_EPOLL_OUT, "W")
+                        << IF_DIRNAME(it->events, SRT_EPOLL_ERR, "E"));
+
+                if (ed.checkEdge(it)) // NOTE: potentially erases 'it'.
+                {
+                    IF_HEAVY_LOGGING(debug_sockets << "!");
+                }
             }
 
+            HLOGC(mglog.Debug, log << "CEPoll::wait: REPORTED " << total << "/" << total_noticed
+                    << debug_sockets.str());
+
             if (lrfds || lwfds)
             {
 #ifdef LINUX
                 const int max_events = ed.m_sLocals.size();
                 epoll_event ev[max_events];
                 int nfds = ::epoll_wait(ed.m_iLocalID, ev, max_events, 0);
 
+                int otal ATR_UNUSED = total;
```

Comment:
Add `const`
"otal" for total without "t"? Maybe `total_old` would be more descriptive?

Reply:
Could be, moreover, I think it could be put under `IF_HEAVY_LOGGING`, as it's logging-only variable.